src/router/injectRoute.ts

```diff
@@ -1,12 +1,12 @@
-// useRouter.ts
 import { inject } from "vue";
 import { Router } from "vue-router";
+import { router as projectRoutes } from "./projectRoutes";
```

Comment:
I think we should export it as `projectRoutes`. I see multiple name clashes above and below and we might as well have the name different.

Reply:
fixed